### ISSUE TITLE
Same inputs = different image output

### ISSUE THREAD

**Issue by CHollman82** [thundaga/SD-webui-txt2img-script#18]
Hi, this script is very useful to me, thanks for making it. The only trouble I'm having is if I hit "generate" and get an image and then doing nothing else hit "generate" again I get a very similar but different image every time... why is this?

So this is my use-case if it helps: I generate many low-res images without using the hires fix option. From those I pick the ones I want to generate with hires fix and put them in a folder. Using your script I select that folder for batch processing and select to take only the seed from the input images. I then enable hires fix in the UI and run the script.

Doing this consecutive times produces different images every time... and they are always different than just doing hires fix to begin with without using your script.

**Comment by OPCOXE** [comment]
Hi @CHollman82,

if you are using [AUTOMATIC1111's Stable Diffusion WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui) with the [lauch option "--xformers"](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings/8fc36525ce0000f708956130eb35c8ee7f3bd66d) then you should try after removing this lauch option.

If not can you check if you can generating the same image twice without the script ?

**Comment by CHollman82** [comment]
Yes, I am using the A1111 WebUI with the --xformers switch, however I can always get the exact same image if I keep all generation parameters the same.

For extensions I am using adetailer, sd-dynamic-prompts, sd-webui-controlnet, and sd-webui-state-manager

When I noticed this I was using the Face-ID Plus IPAdapter model for ControlNET unit 0 and Tile - Resample for ControlNET unit 1. I'm also using a variety of LoRA's and TI Embeds... but again without using your script I can always generate the same image repeatedly if I don't change any of the settings.

It's not a huge deal to me, like I said I love your script and it's very useful, the small differences I get with it are more of a curiosity to me and I thought you'd like to know about it but I'm not like dying for you to fix it or anything. Thank you!

**Comment by OPCOXE** [comment]
As far as I know, you can't have the same image with the "--xformers" launch option. They are always a bit different :
![test](https://github.com/thundaga/SD-webui-txt2img-script/assets/94632173/6b51ffb0-cb94-425f-a9b4-7636bf1929e0)

Can you try without using the ControlNET ?
Since I don't see any update on the repo since AUTOMATIC1111 added the ControNET; I think the script is incompatible with it.
